src/test/unit/controllers/CaseReferenceNumberSearchController.test.ts

```diff
@@ -0,0 +1,129 @@
+import sinon from 'sinon';
+import { Response } from 'express';
+import CaseReferenceNumberSearchController from '../../../main/controllers/CaseReferenceNumberSearchController';
+import fs from 'fs';
+import path from 'path';
+import {mockRequest} from '../mocks/mockRequest';
+import {HearingRequests} from '../../../main/resources/requests/hearingRequests';
+
+const subscriptionCaseSearchController = new CaseReferenceNumberSearchController();
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/subscriptionCaseList.json'), 'utf-8');
+const subscriptionCaseResult = JSON.parse(rawData);
+const stub = sinon.stub(HearingRequests.prototype, 'getHearingByCaseReferenceNumber');
+
+const validCaseNo = 'ABC12345';
+
+describe('Subscription Case Search Controller', () => {
+  let i18n = {};
+  it('should render the search page', () => {
+
+    i18n = {
+      'subscription-urn-search': {},
```

Comment:
this doesnt seem in the right place... just remove this line pls

Reply:
removed.